pydicom/filewriter.py

```diff
@@ -515,7 +517,7 @@ def write_dataset(fp, dataset, parent_encoding=default_encoding):
     """
     _harmonize_properties(dataset, fp)
 
-    if not fp.is_implicit_VR and not dataset.is_original_encoding:
```

Comment:
Maybe the method comment shall also be adapted - it mentions explicit encoding.

Reply:
Done